windows/Makefile/Makefile

```
@@ -0,0 +1,11 @@
+
+VERSION := 3.4.3
+DIR := python-setup
+URL := https://www.python.org/ftp/python/$(VERSION)
+PY_32 := python-$(VERSION).msi
+PY_64 := python-$(VERSION).amd64.msi
+
+all:
+	aria2c --allow-overwrite -d $(DIR) $(URL)/$(PY_32)
+	aria2c --allow-overwrite -d $(DIR) $(URL)/$(PY_64)
```

Here is a suggestion (draft, untested) to use wget and then do a checksum verification:

```suggestion
	wget -r -c --tries=20 -O $(DIR)/$(PY_32) $(URL)/$(PY_32)
	wget -r -c --tries=20 -O $(DIR)/$(PY_64) $(URL)/$(PY_64)
	echo ff19e3f8bde936457b8e53c825110987 $(DIR)/$(PY_32) | md5sum --check -
	echo ff19e3f8bde936457b8e53c825110987 $(DIR)/$(PY_64) | md5sum --check -
```

You will need of course to replace the checksums in the example with the correct ones from a download that you check locally.